lib/tasks/check_content_consistency.rake

```diff
@@ -0,0 +1,29 @@
+namespace :check_content_consistency do
+  def check_content(content_id, locale)
+    checker = ContentConsistencyChecker.new(content_id, locale)
+    errors = checker.call
+
+    if errors.any?
+      puts "#{content_id} 😱"
+      puts errors
+    end
+
+    errors.none?
+  end
+
+  desc "Check documents for consistency with the router-api and content-store"
+  task :one, [:content_id, :locale] => [:environment] do |_, args|
+    content_id = args[:content_id]
+    locale = args[:locale] || "en"
+    check_content(content_id, locale)
+  end
+
+  desc "Check all the documents for consistency with the router-api and content-store"
+  task all: :environment do
+    documents = Document.pluck(:content_id, :locale)
```

Comment:
We've measured the RAM consumption of this as 56MB on the current integration data and therefore don't think this will be an issue.